src/main/views/user-details.njk

```diff
@@ -19,7 +20,19 @@
   ]) }}
 {% endblock %}
 
+{% set eJudiciaryNotification %}
+  <p class="govuk-notification-banner__heading">
+    Please check with the eJudiciary support team to see if there are related accounts.
```

Comment:
I guess if we want to change this text we'd just do a release? (It's not paramaterized)

Reply:
Yeah the project is not internationalised, all the text is hard coded. I would have to redo a fair amount of the project for this to be reasonably manageable in a different way other than this.

Reply:
I have a plan